### PR TITLE
feat: 30 Create a npm script to generate a new rule boilerplate

### PR DESCRIPTION
**Pull request by bilmans**
Adds a new command "yarn generate" that generate :
- A rule file boilerplate
- A test file boilerplate
- A doc file boilerplate

🎯 *Issue* https://github.com/hokla-org/eslint-plugin-custom-rules/issues/30 

## 📣 When merging !

Please inform everyone to upgrade the hokla eslint plugin to benefit from your piece !
https://hokla.slack.com/archives/C02CZUN4QUW
Tell them to run `yarn upgrade @hokla/eslint-plugin-custom-rules` or `npm update @hokla/eslint-plugin-custom-rules`

### CODE REVIEW

**Comment by bilmans** [comment]
Does the script needs to be executable by other devs for it to work (using the `chmod` command) ?

File: generate/generate-boilerplate.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Deactivate rule to prohibiting "require" command
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const fs = require('fs');
+const readline = require('readline');
+const path = require('path');
```
Comment (Ludo171) [inline generate/generate-boilerplate.ts:6]: Il doit y avoir moyen d'accepter les imports classiques 
import * as fs from 'fs'; ...

File: package.json [hunk 2]
```diff
@@ -36,7 +36,8 @@
     "typecheck": "tsc --noEmit",
     "check:eslint-docs": "eslint-doc-generator --check",
     "create:eslint-docs": "eslint-doc-generator --init-rule-docs",
-    "update:eslint-docs": "eslint-doc-generator"
+    "update:eslint-docs": "eslint-doc-generator",
+    "generate": "node generate/generate-boilerplate.ts && yarn build && yarn create:eslint-docs"
```
Comment (Ludo171) [inline package.json:40]: On peut nommer le script plus clairement : 
`rule:generate`

**Review by Ludo171 — CHANGES_REQUESTED** [review]
Très cool ! 

Autres retours de nommage : 
- Le nommage utiliser plutôt du camelCase que du snakeCase. `generateNewRule` `rule.boilerplate.ts` `ruleTests.boilerplate.ts`

**Pull request merged**